WordPress/src/main/java/org/wordpress/android/ui/reader/services/ReaderUpdateService.java

```diff
@@ -200,9 +200,15 @@ private static ReaderTagList parseTags(JSONObject jsonObject, String name, Reade
             String internalName = it.next();
             JSONObject jsonTopic = jsonTopics.optJSONObject(internalName);
             if (jsonTopic != null) {
-                String tagName = JSONUtils.getStringDecoded(jsonTopic, "title");
+                String tagTitle = JSONUtils.getStringDecoded(jsonTopic, "title");
+                String tagName;
+                if (jsonTopic.has("display_name")) {
+                    tagName = JSONUtils.getStringDecoded(jsonTopic, "display_name");
+                } else {
+                    tagName = tagTitle;
+                }
                 String endpoint = JSONUtils.getString(jsonTopic, "URL");
-                topics.add(new ReaderTag(tagName, endpoint, topicType));
+                topics.add(new ReaderTag(tagName, tagTitle, endpoint, topicType));
```

Comment:
Maybe check to ensure the `tagTitle` is capitalized the way we like, then store the capitalized title.